.github/workflows/build.yml

```diff
@@ -74,14 +95,90 @@ jobs:
         ./run_example.sh --dry-run simple_app_under_sev setup_with_auto_policy_generation_for_SEV
 
         #! ---------------------------------------------------------------------
-        #! Test: Execute script to compile, build and run simple_app.
         #! This will also check that utilities programs still compile
+        echo " "
+        echo "******************************************************************"
+        echo "* Test: Execute script to compile, build and run simple_app."
+        echo "******************************************************************"
+        echo " "
         ./cleanup.sh
         ./run_example.sh simple_app
         ./cleanup.sh
 
-        #! ---------------------------------------------------------------------
-        #! Few other miscellaneous checks
-        cd ../certifier_service/oelib
+        popd
+
+        echo " "
+        echo "*************************************************************************************"
+        echo "* Build and install SEV-SNP simulator, to run Cert tests with simulated SEV-enabled."
+        echo "*************************************************************************************"
+        echo " "
+        pushd ./sev-snp-simulator
+
+        make clean
+        make
+        make keys
+
+        #! You may need this in your dev-box, but on CI this will not be needed
+        #! as we have not run 'insmod', yet.
+        #! make rmmod
+
+        make insmod
+
+        echo " "
+        echo "******************************************************************"
+        echo "* Run sev-test ... "
+        echo "******************************************************************"
+        echo " "
+
+        pushd ./test
+        make sev-test
+        sudo ./sev-test
+
+        popd
+        popd
+
+        echo " "
+        echo "******************************************************************"
+        echo "* Check that Certifier tests run clean with simulated SEV-enabled."
+        echo "******************************************************************"
+        echo " "
+
+        pushd src
+        make -f certifier_tests.mak clean
+        ENABLE_SEV=1 make -f certifier_tests.mak
+        sudo ./certifier_tests.exe --print_all=true
+
+        echo " "
+        echo "******************************************************************"
+        echo "* Check that Certifier builds with simulated SEV-enabled."
+        echo "******************************************************************"
+        echo " "
+        make -f certifier.mak clean
+        make -f certifier_tests.mak clean
+        ENABLE_SEV=1 make -f certifier.mak
+        popd
+
+        #! Run script that will setup s/w required to build Policy Generator for SEV-app
+        ./CI/scripts/setup-JSON-schema-validator-for-SEV-apps.sh
```

Comment:
Added a new `CI/scripts` dir with one setup script. This packages all the steps required.